Atizaz abid

Atizaz-Abid opened the pull request:
Added the models and dash board and get categories api

Commit:
models

Atizaz-Abid committed:
Delete models.py

Commit:
models01

Atizaz-Abid committed:
Add files via upload

Atizaz-Abid committed:
Add files via upload

tahirawan4 commented on models.py:
```diff
@@ -0,0 +1,151 @@
+class TimeStampedModel(models.Model):
+    created = models.DateTimeField(auto_now_add=True)
+    modified = models.DateTimeField(auto_now=True)
+    isRemove = models.BooleanField(default=False)
```
Always use small letters with `_`. Variable name must be `is_remove`

tahirawan4 commented on models.py:
```diff
@@ -0,0 +1,151 @@
+                               help_text="Upload your photo for Avatar")
+    about = models.TextField()
+    website = models.URLField(max_length=200, blank=True, null=True)
+    isRemove = models.BooleanField(default=False)
```
Same as above

tahirawan4 commented on models.py:
```diff
@@ -0,0 +1,151 @@
+
+    class Meta:
+        verbose_name = 'Detail Author'
+        verbose_name_plural = 'Authors'
```
We don't need these meta variables, as author plural is authors by default. Remove it please

tahirawan4 commented on models.py:
```diff
@@ -0,0 +1,151 @@
+    def get_total_posts(self):
+        return Feeds.objects.filter(tags__pk=self.pk).count()
+
+    class Meta:
```
We won't need here too.

tahirawan4 commented on models.py:
```diff
@@ -0,0 +1,151 @@
+    api_url = models.TextField(null=True)
+    description = models.TextField(null=True)
+    time = models.ForeignKey(TimeStampedModel,on_delete=models.CASCADE,related_name='time_stamp',db_index=True)
+    isRemove = models.BooleanField(default=False)
```
Update this one.

tahirawan4 commented on models.py:
```diff
@@ -0,0 +1,151 @@
+
+    publish = models.BooleanField(default=True)
+    objects = FeedsApiUrl.as_manager()
+    isRemove = models.BooleanField(default=False)
```
Update it

tahirawan4 commented on models.py:
```diff
@@ -0,0 +1,151 @@
+    slug = models.SlugField(max_length=200, unique=True)
+    description = RedactorField()
+    publish = models.BooleanField(default=True)
+    isRemove = models.BooleanField(default=False)
```
Update it

tahirawan4 commented on models.py:
```diff
@@ -0,0 +1,151 @@
+
+    class Meta:
+        verbose_name = "Detail Page"
+        verbose_name_plural = "Pages"
```
We won't need this.

tahirawan4 commented on models.py:
```diff
@@ -0,0 +1,151 @@
+
+    class Meta:
+        verbose_name = 'Detail Gallery'
+        verbose_name_plural = 'Galleries'
```
This is good.

tahirawan4 commented on models.py:
```diff
@@ -0,0 +1,151 @@
+class Visitor(TimeStampedModel):
+    post = models.ForeignKey(Post, related_name='post_visitor')
+    ip = models.CharField(max_length=40)
+    isRemove = models.BooleanField(default=False)
```
Update it.

tahirawan4 commented on models.py:
```diff
@@ -0,0 +1,151 @@
+
+    class Meta:
+        verbose_name = 'Detail Visitor'
+        verbose_name_plural = 'Visitors'
```
Remove it.

tahirawan4 commented on parser/constants.py:
```diff
@@ -0,0 +1,16 @@
+
+
+
```
Update all constants with all capitals.

tahirawan4 commented on parser/models.py:
```diff
@@ -0,0 +1,151 @@
+from django.db import models
```
Why we have two models files here?
Please remove one and update all the migrations.

tahirawan4 commented on parser/views.py:
```diff
@@ -2,7 +2,48 @@
+
+
+def get_categories(request):
+    # TODO: This view is call from the url and will return the Feedly fetch records to html view
```
Why a lot of line breaks here??

tahirawan4 reviewed (changes requested):
Please remove all `.`  files committed in the repo.

Atizaz-Abid commented:
ok
thanks for the review
i will update ,but let me know these model are enough for the v1

On Sun, Sep 10, 2017 at 9:21 PM, Tahir Fazal <notifications@github.com>
wrote:

> *@tahirawan4* requested changes on this pull request.
>
> Please remove all . files committed in the repo.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/tahirawan4/Martium/pull/2#pullrequestreview-61688613>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AOHSqYSxNVIueYYUb-6eq4el3G8DtiSGks5shAx8gaJpZM4PQeMA>
> .
>

Atizaz-Abid committed:
Delete models.py

Atizaz-Abid committed:
Update models.py

All the required changes are done
updated the variables added _ in
removed the Tag model and visitors model
I think Meta tags are help in future

Atizaz-Abid committed:
Update constants.py

changes all the constants to caps

Atizaz-Abid committed:
Update views.py

updated the constants

tahirawan4 commented:
@aitzazabid @AhmedKakar 
Please resolve conflicts by merging master into this branch,
How to do that.
1) Change your branch to master
2) Pull master
3) Switch Back to this branch
4) And now merge master in this branch

Also models and not good for V1, I need to update the, too. I am exploring feedly. As soon as i got something i will update it.

Atizaz-Abid committed:
Update constants.py

changes the url
replace sandbox to sandbox7

Commit:
'merge'

Commit:
Merge branch 'atizaz_abid' of https://github.com/tahirawan4/Martium into atizaz_abid

# Conflicts:
#	parser/models.py

Commit:
'added'

tahirawan4 commented:
Where are the migrations??? Did you try to run this code?? @aitzazabid @AhmedKakar

tahirawan4 commented on parser/models.py:
```diff
@@ -0,0 +1,119 @@
+                              null=True,
+                              blank=True,
+                              help_text='Optional cover post')
+    description = RedactorField()
```
@aitzazabid  @AhmedKakar  What is this field? How do you import this field??

tahirawan4 commented on parser/models.py:
```diff
@@ -0,0 +1,119 @@
+    author = models.ForeignKey(Author, related_name='author_page')
+    title = models.CharField(max_length=200)
+    slug = models.SlugField(max_length=200, unique=True)
+    description = RedactorField()
```
Same here?

tahirawan4 commented:
@aitzazabid @AhmedKakar As you are using requests in views, When ever you use a third party module, please add it into requirements file.

Atizaz-Abid commented:
ok
we didn't run the migration yet ,waiting for the best and approved models

On Tue, Sep 12, 2017 at 2:15 AM, Tahir Fazal <notifications@github.com>
wrote:

> Reopened #2 <https://github.com/tahirawan4/Martium/pull/2>.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/tahirawan4/Martium/pull/2#event-1244134580>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/AOHSqaCRhCOmuJ3kJ3f_gWt7MMseOeO4ks5shaLlgaJpZM4PQeMA>
> .
>

tahirawan49 committed:
Updating constants

tahirawan4 committed:
Merge branch 'master' into atizaz_abid
